src/Components/UI/Modal/Login.jsx

```diff
@@ -0,0 +1,108 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useModal } from "../../../Context";
+import Button from "../Button/Button";
+import InputTypeOne from "../Input/InputTypeOne";
+import "./Login.css";
+
+const Login = () => {
+  const [loginInput, setLoginInput] = useState({
+    email: "",
+    password: "",
+  });
+
+  const { setShowLogin, setShowSignup } = useModal();
+  const navigate = useNavigate();
+
+  const onLoginSubmitHandler = (e) => {
```

Comment:
instead of e, may be better naming like "modal event" could be provided. This is also fine. Just a suggestion.

Reply:
Ok.